universal-login-react/src/core/utils/calculateTransactionFeeInUSD.ts

```diff
@@ -0,0 +1,7 @@
+import {utils} from 'ethers';
+import {BigNumberish} from 'ethers/utils';
+import {safeMultiply} from '@universal-login/commons';
+import {formatCurrency} from './formatCurrency';
+
+export const calculateTransactionFeeInUSD = (usdAmount: BigNumberish, gasLimit: BigNumberish) =>
+  formatCurrency(safeMultiply(utils.parseEther(usdAmount.toString()), gasLimit));
```

Comment:
Would `calculateTransactionFeeInEUR` be different?
Consider rename to `calculateTransactionFee` ?